react_on_rails_pro/LICENSE_SETUP.md

```
@@ -113,17 +113,147 @@ CI/CD environments work without a license. If your CI pipeline deploys to produc
+
+```bash
+# Human-readable output
+bundle exec rake react_on_rails_pro:verify_license
+
+# JSON output (for CI/CD and scripting)
+FORMAT=json bundle exec rake react_on_rails_pro:verify_license
+```
+
+**Example text output:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)